openedx/features/enterprise_support/tests/mixins/enterprise.py

```diff
@@ -57,6 +57,18 @@ def mock_enterprise_course_enrollment_post_api_failure(self):  # pylint: disable
             status=500
         )
 
+    def mock_enterprise_course_enrollment_get_api(self, **kwargs):
+        result = {
+            'results': [{**kwargs}] if kwargs else []
```

Comment:
Are there use cases we are covering which would return an empty set for `results`?  Not clear from the tests that are being added, but maybe there's something else already in place.